packages/cli/Readme.md

```diff
@@ -0,0 +1,57 @@
+## Webstudio CLI
+
+The Webstudio CLI is designed to help you seamlessly link, sync, and build projects from your web development workspace. This README will guide you through the process of setting up linking, syncing, building, and running your projects using the Webstudio CLI commands.
```

Comment:
> from your web development workspace

don't know what that means, sounds like its about checking out the entire repo but this is not the goal for the cli, somebody should be able to use the CLI without even downloading the builder repostority

Reply:
unless you meant the "project folder" something ...